week-15/quizzes/03-reducers.md

```diff
@@ -0,0 +1,92 @@
+# Reducers Quiz
+
+This quiz is to test you on what reducers are and how to use them.
+
+## The questions
+
+<quiz>
+  <question>
+    <p>
+      The purpose of a reducer function is to:
+    </p>
+    <answer>Reduce the `state` object to a single value</answer>
+    <answer correct>Update `state`</answer>
+    <answer>Make a new `state` object</answer>
+    <answer>Dispatch an `action`</answer>
+    <explanation>
+      The Redux store has a reducer function for updating the current state and 
+      returning that new state. 
+    </explanation>
```

Comment:
I think the explanation for this is a little confusing since it mentions both "updating the current state" and "returning that new state" which could be understood as "making a new state". Perhaps something along the lines of "returning an updated version of the state" as an answer choice would work a little better with this explanation.